### DISCUSSION TITLE
some error about udf implement

### DISCUSSION THREAD

**Discussion by daikon12** [apache/doris#11544] [A - General / Q&A] [answered]
There is a error reported when I create a udf named belong.
ERROR 1105 (HY000): errCode = 2, detailMessage = VARCHAR size must be > 0: -1

The general create content ：
CREATE FUNCTION belong(VARCHAR,VARCHAR,VARCHAR,...) RETURNS BOOLEAN PROPERTIES (
"symbol" =  "_ZN9doris_udf6BelongEPNS_15FunctionContextERKNS_9StringValES4_iPS3_",
"name" = "demo.belong",
"object_file" = "https:XXXXXXX"
);

The function definition  in udf_belong.h   ：
BooleanVal Belong(FunctionContext* ,const StringVal& str,const StringVal& sep,int num_in, const StringVal* in_strs);

Please tell me the reason for the error and the solution。

extremely grateful ！

**Comment by ChPi** [comment, marked as answer]
try belong(VARCHAR(256) instead of belong(VARCHAR

**Reply by daikon12** [reply]
thanks for the guide, it worked.
